ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.9.yaml

```diff
@@ -93,7 +93,7 @@ tests:
       BASE_DOMAIN: shiftstack.devcluster.openshift.com
       CONFIG_TYPE: minimal
       E2E_RUN_TAGS: '@openstack-upi'
-      TAG_VERSION: '@4.10'
+      TAG_VERSION: '@4.9'
     post:
     - chain: ipi-openstack-post
```

Comment:
Wrong installation step used there, https://github.com/openshift/release/pull/22959/files#diff-44c8929c9f4e8612bcc7b128e0a5e48f17298061be14ffcc0d344893e676d009
We should use UPI installation for UPI tests. But IPI installation is used here.
This is a know issue, which needs more tests. I'll check/fix that in a separate PR.